perun-core/src/main/java/cz/metacentrum/perun/core/impl/AuthzResolverImpl.java

```diff
@@ -174,15 +174,30 @@ public void initialize() throws InternalErrorException {
 		}
 	}
 
-	public static List<Role> getRolesWhichCanWorkWithAttribute(ActionType actionType, AttributeDefinition attrDef) throws InternalErrorException {
-		String actType = actionType.getActionType().toLowerCase();
-		try {
-			return jdbc.query("select distinct roles.name from attributes_authz " +
-					"join roles on attributes_authz.role_id=roles.id " +
-					"join action_types on attributes_authz.action_type_id=action_types.id " +
-					"where attributes_authz.attr_id=? and action_types.action_type=?", AUTHZROLE_MAPPER_FOR_ATTRIBUTES, attrDef.getId(), actType);
+	public static Map<Role, Set<ActionType>> getRolesWhichCanWorkWithAttribute(ActionType actionType, AttributeDefinition attrDef) throws InternalErrorException {
+		String actType = actionType.getActionType().toLowerCase() + "%";
+		try {
+			List<Pair<Role, ActionType>> pairs = jdbc.query("select distinct roles.name, action_types.action_type from attributes_authz " +
+							"join roles on attributes_authz.role_id=roles.id " +
+							"join action_types on attributes_authz.action_type_id=action_types.id " +
+							"where attributes_authz.attr_id=? and action_types.action_type like ?",
```

Comment:
JavaDoc should be updated for this method in BLImpl. It is not straightforward why there is "like" instead of "=". Also in old javaDoc return type is List instead of Map.